uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/daemon/ssh/GitSSHService.java

```diff
@@ -51,26 +59,41 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(GitSSHService.class);
 
-    private final SshServer sshd = buildSshServer();
-
+    private static List<BuiltinCiphers> managedCiphers =
+            Collections.unmodifiableList(Arrays.asList(
+                    BuiltinCiphers.aes128ctr,
+                    BuiltinCiphers.aes192ctr,
+                    BuiltinCiphers.aes256ctr,
+                    BuiltinCiphers.arcfour256,
+                    BuiltinCiphers.arcfour128,
+                    BuiltinCiphers.aes192cbc,
+                    BuiltinCiphers.aes256cbc
+            ));
+
+    private static List<BuiltinMacs> managedMACs =
```

Comment:
not big fan of static here, but at least add final to it

Reply:
Ok

Reply:
Ok